website/public/js/services/groupServices.js

```diff
@@ -77,6 +77,21 @@
         $location.path("/options/groups/party");
       }
     }
+    
+    function isOverPartyCap(group) {
```

Comment:
Isn't this just `group.isOverPartyCap` as defined on the virtual attribute for the group instance? Or am I missing something?